packages/core/src/components/Dropdown/Dropdown.styles.tsx

```diff
@@ -37,7 +37,7 @@ export const StyledDropdown = styled(
     $readOnly?: boolean;
   }) => ({
```

Comment:
The dropdown in DS3 doesn't have the right styling:

<img width="407" alt="Screenshot 2023-06-16 at 12 13 18" src="https://github.com/lumada-design/hv-uikit-react/assets/43220251/f756b6d6-b202-405f-855f-5aeb1c93e978">

Before:

<img width="407" alt="Screenshot 2023-06-16 at 12 16 07" src="https://github.com/lumada-design/hv-uikit-react/assets/43220251/de950236-8f71-4595-bb66-7002c2e3e729">

Reply:
Fixed. `HvDrodown` still had some unnecessary overrides